docs/marionette.region.md

```diff
@@ -29,81 +29,67 @@ Using the `LayoutView` class you can create nested regions.
-named regions and either jQuery selectors or `Region` objects. You may
-call this method as many times as you like, and it will continue adding regions
-to the app.
+The Application defines a single region using the `region` attribute, and access
+it using `getRegion`, or `showView`. Below is a short example:
 
 ```js
-MyApp.addRegions({
-  mainRegion: "#main-content",
-  navigationRegion: "#navigation"
-});
-```
-
-As soon as you call `addRegions`, your regions are available on your
-app object. In the above, example `MyApp.mainRegion` and `MyApp.navigationRegion`
-would be available for use immediately.
-
-If you specify the same region name twice, the last one in wins.
+var Marionette = require('backbone.marionette');
 
-You can also add regions via `LayoutView`s:
+var App = Marionette.Application.extend({
+  region: '#main-content',
 
-```js
-var AppLayoutView = Marionette.LayoutView.extend({
-  template: "#layout-view-template",
-
-  regions: {
-    menu: "#menu",
-    content: "#content"
+  onStart: function() {
+    var main = this.getRegion();
   }
 });
-var layoutView = new AppLayoutView();
-layoutView.render();
-// you can access your region by its name
-layoutView.menu.show(new MenuView());
-// also with the getRegion method
-layoutView.getRegion('menu').show(new MenuView());
-// or in this case reference and show it with the showChildView method
-layoutView.showChildView('content', new MainContentView())
 ```
 
-### Region Configuration Types
+For more information, see the
+[Application docs](./marionette.application.md#root-layout).
 
-Marionette supports multiple ways to define regions on your `Application` or `LayoutView`.
+## Region Configuration Types
 
-#### String Selector
+Marionette supports multiple ways to define regions on your `Application` or
+`View`. This section will document the different types as applied to `View`,
+although they will work for `Application` as well - just replace `regions` with
+`region` in your definition.
+
+### String Selector
 
 You can use a jQuery string selector to define regions.
 
 ```js
-App.addRegions({
-  mainRegion: '#main'
+var Marionette = require('backbone.marionette');
+
+var MyView = Marionette.View.extend({
+  regions: {
+    mainRegion: '#main'
+  }
 });
 ```
 
-#### Region Class
-
-If you've created a custom region class, you can use it to
-define your region.
+### Region Class
 
-**NOTE:** Make sure the region class has an `el`
-property set or it won't work!
+If you've created a custom region class with an `el` property set, you can pass
+it in as an argument to your `regions` constructor.
 
 ```js
+var Marionette = require('backbone.marionette');
+
 var MyRegion = Marionette.Region.extend({
-  el: '#main-nav'
+  el: '#main-nav'  // This is mandatory
 });
 
-App.addRegions({
-  navigationRegion: MyRegion
+var MyView = Marionette.View.extend({
+  regions: {
+    mainRegion: MyRegion
+  }
 });
 ```
 
-#### Object Literal
+### Object Literal
```

Comment:
This section needs some cleaning up at some point.  Not supported at all yet too until your issue gets in.. this is also another case where we're doing quite a bit of describing region functionality.